hedera-mirror-rest/model/customFeeLimits.js

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: Apache-2.0
+
+import {proto} from '@hashgraph/proto';
+import FixedFee from './fixedFee.js';
+
+class CustomFeeLimits {
+  /**
+   * Parses an array of serialized CustomFeeLimit messages from transaction.max_custom_fees
+   * @param {Buffer[]} customFeeLimits - An array of byte arrays representing serialized CustomFeeLimit messages.
+   */
+  constructor(customFeeLimits) {
+    if (!Array.isArray(customFeeLimits) || customFeeLimits.length === 0) {
+      this.fees = [];
+      return;
+    }
+    this.fees = (customFeeLimits ?? []).map((feeBytes) => {
```

Comment:
null and empty array check is already done.

```suggestion
    this.fees = customFeeLimits.map((feeBytes) => {
```